### PR TITLE
Resolve conflicts

### PR DESCRIPTION
**Pull request by jiwalker-usgs**
PTs changes along with conflicts fixed and debugger removed.

### CODE REVIEW

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java [hunk 1]
```diff
@@ -105,30 +105,26 @@ public Response searchItems(
...
-				}
-				else {
-					Map<String, Object> ok = new HashMap<String, Object>() {
-						private static final long serialVersionUID = 2398472L;
+		try (ItemManager itemManager = new ItemManager()) {
+			Item item = Item.fromJSON(content);
+			final String id = itemManager.persist(item);
+			if (null == id) {
+				throw new BadRequestException();
+			}
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java]: oh my god. carriage return after the curly? SERIOUSLY?!

File: pom.xml [hunk 2]
```diff
@@ -420,20 +420,30 @@
...
+			    <groupId>org.glassfish.jersey.ext</groupId>
+			    <artifactId>jersey-mvc-jsp</artifactId>
+			    <version>2.16</version>
 			</dependency>
 			<dependency>
 				<groupId>com.vividsolutions</groupId>
 				<artifactId>jts</artifactId>
 				<version>1.13</version>
 			</dependency>
+	        <dependency>
```
Comment (isuftin) [inline pom.xml]: omg tabbing

Comment (thongsav-usgs) [inline pom.xml]: Are we still not allowed to talk about that?

Comment (isuftin) [inline pom.xml]: tabs. no spaces. But this is tab aligned with the outer node

**Pull request merged**